R/meta_regression/regression_plot_panel.R

```diff
@@ -0,0 +1,276 @@
+.CreateInlineBlock <- function(..., style = NULL) {
+  return(
+    div(
+      ...,
+      style = paste0("display: inline-block; ", style)
+    )
+  )
+}
+
+.AddTooltip <- function(..., tooltip) {
+  return(
+    div(
+      ...,
+      title = tooltip
+    )
+  )
+}
+
+#' Create the regression plot panel.
+#'
+#' @param id ID of the module.
+#' @return Div containing the module UI.
+regression_plot_panel_ui <- function(id) {
+  ns <- NS(id)
+  
+  sidebarLayout(
+    position = "right",
+    mainPanel = mainPanel(
+      width = 9,
+      plotOutput(outputId = ns("regression_plot"), height = "800px")
+    ),
+    sidebarPanel = sidebarPanel(
+      width = 3,
+      # Add comparators
+      selectInput(inputId = ns("add_comparator_dropdown"), label = "Add Comparator", choices = c(), selectize = FALSE),
+      actionButton(inputId = ns("add_comparator_btn"), label = "Add to plot"),
+      div(
+        style = "float: right;",
+        actionButton(inputId = ns("add_all_btn"), label = "Add all")
+      ),
+      br(),
+      br(),
+      # Remove comparators
+      selectInput(inputId = ns("remove_comparator_dropdown"), label = "Remove Comparator", choices = c(), selectize = FALSE),
+      actionButton(inputId = ns("remove_comparator_btn"), label = "Remove"),
+      div(
+        style = "float: right;",
+        actionButton(inputId = ns("remove_all_btn"), label = "Remove all")
+      ),
+      
+      h3("Plot Options"),
+      # Covariate value
+      .AddTooltip(
+        checkboxInput(inputId = ns("covariate"), label = "Show covariate value", value = TRUE),
```

Comment:
I really like the tooltips but it isn't obvious that they are there. Is there any way of making it more obvious without having a messy looking ui?

Reply:
I've added tooltip icons to make it a little more clear.